src/AddTodo/AddTodoForm.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState } from 'react';
+import { TodoData } from '../types/Todo';
+
+interface Props {
+  setHasTitleError: (arg: boolean) => void
+  addTodo: (todoData: TodoData) => void
+  userId: number
+}
+
+export const AddTodoForm: React.FC<Props> = ({
+  setHasTitleError,
+  addTodo,
+  userId,
+}) => {
+  const [title, setTitle] = useState('');
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!title) {
+      setHasTitleError(true);
+
+      return;
+    }
```

Comment:
To avoid any issues with whitespace todos, use trim()
```suggestion
    if (!title.trim()) {
      setHasTitleError(true);

      return;
    }
```

Reply:
In working example you may also add todo with whitespace that's why i left it